server/api/rounds.py

```diff
@@ -175,8 +176,8 @@ def is_round_ready_to_finish(election: Election, round: Round) -> bool:
         return num_jurisdictions_without_results == 0
 
 
-def is_audit_complete(round: Round):
-    if not round.ended_at:
+def is_audit_complete(round: Optional[Round]):
+    if not (round and round.ended_at):
```

Comment:
In what cases will a `round` not exist? If an audit has no rounds, does that mean it hasn't started yet and should be considered incomplete?

Reply:
No rounds means the audit hasn't started yet (is still in the setup phase). So yes, I'm considering that incomplete.